src/gui/updownratiodlg.cpp

```diff
@@ -28,13 +28,16 @@
  * Contact : chris@qbittorrent.org
```

Comment:
@naikel:  Mostly irrelevant to this PR, but I am pretty sure you should remove this line and edit Line 3 of this file to read:  
`  * Copyright (C) 2011  Christian Kandeler, Christophe Dumez <chris@qbittorrent.org>`.  

Newer code uses that format, and I have seen @glassez change existing files to it in other PRs.